runtime/binding-http-filesystem/src/main/java/io/aklivity/zilla/runtime/binding/http/filesystem/internal/config/HttpFileSystemBindingConfig.java

```diff
@@ -27,13 +26,8 @@
 
 public final class HttpFileSystemBindingConfig
 {
-    private static final Predicate<HttpHeaderFW> HEADER_NAME_PATH;
-
-    static
-    {
-        String8FW headerPath = new String8FW(":path");
-        HEADER_NAME_PATH = h -> headerPath.equals(h.name());
-    }
+    private static final String8FW HEADER_METHOD_NAME = new String8FW(":method");
+    private static final String8FW HEADER_NAME_PATH = new String8FW(":path");
```

Comment:
Please change these back to predicates, otherwise the lambda has overhead per call to resolve when defined anonymously.

Also please make sure the constants are named consistently, see placement of `_NAME`.